src/components/homePage/Testimonials.vue

```diff
@@ -1,13 +1,41 @@
 <template>
-<div id="testimonials">
-	<h2>Testimonials</h2>
-	<!-- SIDE SCROLLER -->
-</div>
+  <div id="testimonials">
+    <h2>Testimonials</h2>
+    <!-- SIDE SCROLLER -->
+    <div>
+      <img src="../../assets/testimonial_building.svg" id="building" />
+      <vueper-slides id="slides" :bullets="false" :arrows="false" autoplay>
+        <vueper-slide 
```

Comment:
There was a miscommunication on my part - the whole white box should scroll to look like it's going behind the building. Each testimonial should be "attached" to its card and swipe with it, if that makes any sense.